dashboard-frontend/src/types/study.ts

```diff
@@ -1,31 +1,36 @@
+import { Currency } from './shared';
+
 export interface Study {
   id: string;
   name: string;
   slug: string;
-  createdAt: string;
+  createdAt: number;
   studyProgressList: StudyProgress[];
   stratumProgressList: StratumProgress[];
 }
 
-interface StudyProgress {
+export interface StudyProgress {
   id: string;
-  datetime: string;
+  datetime: number;
+  desiredParticipants: number;
   currentParticipants: number;
   expectedParticipants: number;
   currentAverageDeviation: number;
   expectedAverageDeviation: number;
 }
 
-interface StratumProgress {
+export interface StratumProgress {
   id: string;
   name: string;
-  datetime: string;
+  datetime: number;
   desiredPercentage: number;
   currentPercentage: number;
   expectedPercentage: number;
   percentageDeviationFromGoal: number;
   desiredParticipants: number;
+  expectedParticipants: number;
   currentParticipants: number;
   currentBudget: number;
   currentPricePerParticipant: number;
-}
\ No newline at end of file
+  currency: Currency;
```

Comment:
@nandanrao Not sure if at the moment we are only dealing with **usd** but I added **currency** since I imagine that we want to have support for other ones.

Reply:
Hrm, yes, but I'm actually not sure if we ever need to know it. Or will know it. Their Facebook Ad Account will be set to the currency and we just deal with numbers...

Reply:
Mmmm oki, I assumed that we had this information and I wanted to use it to display the currency symbol in the UI alongside **currentBudget** and **currentPricePerParticipant**

But you're right, thinking again about it I don't think this is needed for this kind of app. This info is more useful for apps that sell something to their users. I'll remove it 👍